apps/wallet/src/ui/app/hooks/useGetCoinBalance.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiAddress } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+
+import { useRpc } from '_hooks';
+type GetCoinBalanceProps = {
+    address?: SuiAddress | null;
+    coinType: string;
+};
+export function useGetCoinBalance({ address, coinType }: GetCoinBalanceProps) {
+    const rpc = useRpc();
+    return useQuery(
+        // combine address and coinType to make a unique key account for multiple addresses and coins
```

Comment:
Just comma separate them, that's what the point of the array is. You can remove this comment too.